lib/Doctrine/DBAL/Platforms/DB2Platform.php

```diff
@@ -878,4 +880,64 @@ protected function getReservedKeywordsClass()
     {
         return Keywords\DB2Keywords::class;
     }
+    
+    /**
+     * Prepare ORDER BY string for OVER() if applicable
+     * 
+     * @param  string $query
+     * 
+     * @return array|string[]|mixed[]
```

Comment:
This doesn't look usable. Why doesn't it always return a string?

Reply:
Good question... Let me fix that...